src/components/common/WorkCard/index.js

```diff
@@ -1,74 +1,28 @@
 import PropTypes from 'prop-types'
-import React, { useRef } from 'react'
+import React from 'react'
 
-import { css } from 'styled-components'
-import { WorkWrapper, TextWrapper } from './styles'
+import { WorkWrapper } from './styles'
 import { CapsHeadingSmall, H4 } from '../../elements'
 
 import Image from '../../image'
 
-const WorkCard = ({
-  smallHeading,
-  bigHeading,
-  fileName,
-  altText,
-  topDestance,
-  cardNumber,
-}) => {
-  const ref = useRef()
-  let light = 0
-  switch (cardNumber) {
-    case 1:
-      light = topDestance > 2794
-      break
-    case 2:
-      light = topDestance > 3115
-      break
-    case 3:
-      light = topDestance > 3426
-      break
-    case 4:
-      light = topDestance > 3730
-      break
-    default:
-      break
-  }
```

Comment:
this code was handling the brightness of the work card is this effect not needed anymore?

Reply:
I think it's not needed at the moment as we have two case studies so was planning on putting it back when we have four